docs/docs/100-reference/01-command-line/acorn_install.md

```diff
@@ -21,6 +21,7 @@ acorn install
 ```
       --acorn-dns string                       enabled|disabled|auto. If enabled, containers created by Acorn will get public FQDNs. Auto functions as disabled if a custom clusterDomain has been supplied (default auto)
       --acorn-dns-endpoint string              The URL to access the Acorn DNS service
+      --allow-traffic-from-namespace strings   Namespaces that can bypass NetworkPolicies and send network traffic to all Acorn apps
```

Comment:
"bypass" sounds off. like theyre circumventing net policies, but its just part of the policy to allow the traffic. How about:

> Namespaces that are allowed to send network traffic to all Acorn apps

Reply:
Good call